devine/core/tracks/track.py

```diff
@@ -542,6 +542,7 @@ def _ffmpeg(extra_args: list[str] = None):
             else:
                 raise
 
+        self.path.unlink()
```

Comment:
Can you make it original_path.unlink() instead?

Reply:
Of course 👍 https://github.com/devine-dl/devine/pull/114/commits/0542e89ad24ba7b94f4d2d1e4fdc5895a4a7c756